src/waitlist.cpp

```diff
@@ -95,36 +134,25 @@ bool WaitingList::clientLogin(const Player* player)
 		return false;
 	}
 
-	slot = priorityWaitList.size();
+	slot = info->priorityWaitList.size();
 	if (player->isPremium()) {
-		priorityWaitList.emplace_back(OTSYS_TIME() + (getTimeout(slot + 1) * 1000), player->getGUID());
+		info->priorityWaitList.emplace_back(OTSYS_TIME() + (getTimeout(slot + 1) * 1000), player->getGUID());
 	} else {
-		slot += waitList.size();
-		waitList.emplace_back(OTSYS_TIME() + (getTimeout(slot + 1) * 1000), player->getGUID());
+		slot += info->waitList.size();
```

Comment:
I was just asking about the queue. Any performance gains or losses are unlikely to have any significant effect here, so w/e is simpler should be used ;).